stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkPeerManager.java

```diff
@@ -50,9 +51,19 @@ public AsyncPrettyPrinterRegistry getAsyncPrettyPrinterRegistry() {
     return mAsyncPrettyPrinterRegistry;
   }
 
+  public void setPrettyPrinterInitializer(AsyncPrettyPrinterInitializer initializer) {
+    mPrettyPrinterInitializer = Util.throwIfNull(initializer);
```

Comment:
throw if `mPrettyPrinterInitializer` is null. Basically only allow `setPrettyPrinterInitializer` to be called once.